src/App.tsx

```diff
@@ -1,14 +1,17 @@
 import { ThemeProvider as MuiThemeProvider } from '@material-ui/core';
 import React from 'react';
 import { ThemeProvider } from 'styled-components';
-import { DemoContainer } from './demo/DemoContainer';
+import { StylesProvider } from '@material-ui/core/styles';
+import { HomePage } from './home-page/HomePage';
 import { theme } from './layout/theme';
 
 export const App: React.FC<{}> = () => (
-  <MuiThemeProvider theme={theme}>
-    <ThemeProvider theme={theme}>
-      {/* Replace with actual content */}
-      <DemoContainer />
-    </ThemeProvider>
-  </MuiThemeProvider>
+  <StylesProvider injectFirst>
+    <MuiThemeProvider theme={theme}>
+      <ThemeProvider theme={theme}>
+        {/* Replace with actual content */}
```

Comment:
I guess we can remove this comment now :)

Reply:
Ok.